core/templates/pages/exploration-player-page/services/extract-image-filenames-from-model.service.spec.ts

```diff
@@ -37,9 +38,16 @@ describe('Extracting Image file names in the state service', () => {
   let sof: SkillObjectFactory;
   let ecs: ContextService;
   let ctms: ContentTranslationManagerService;
-  let explorationDict;
-  let ImageFilenamesInExploration;
-  let skillDict;
+  let explorationDict: ExplorationBackendDict;
+  let ImageFilenamesInExploration: { [x: string]: string[];
+    'State 1': string[];
+    'State 3': string[];
+    'State 4': string[];
+    'State 5': string[];
+    'State 6': string[];
+    'State 7': string[];
```

Comment:
Done

Reply:
Wait looking at it now, you define generic `[x: string]: string[];`, why you also need to define the specific ones?

Reply:
You're right. The defined generic `[x: string]: string[];` is enough so I removed the specific strings that I defined and it works just as fine.